api/server.js

```diff
@@ -82,8 +82,9 @@ process
     serverUtils.serverError(err, req, res);
   });
 
-  app.listen(apiPort, () => {
+  const server = app.listen(apiPort, () => {
     logger.info('Medic API listening on port ' + apiPort);
   });
-
+  const ONE_HOUR = 60 * 60 * 1000;
+  server.setTimeout(ONE_HOUR);
```

Comment:
How will we know how often the timeout is occurring? Is it worth adding a callback or `timeout` event handler to log this?

Reply:
Timed out requests still appear in logs, and they look like: 
```
medic-api-logs | [2021-07-08 21:00:41] REQ a4723b9f-ac2d-43ec-83d5-9e9acad797de <IP> - GET /medic/_all_docs?key=%22_design%2Fmedic-client%22 HTTP/1.1
medic-api-logs | [2021-07-08 21:02:41] RES a4723b9f-ac2d-43ec-83d5-9e9acad797de <IP> - GET /medic/_all_docs?key=%22_design%2Fmedic-client%22 HTTP/1.1 - - - ms
```

However interrupted requests also "terminate" alike, with no status, content-length or response time logged. The telling sign is the difference between the REQ and RES entry (which here is 2 minutes because this is a prod log). 

Do you think an additional entry with the timeout would be required? Maybe we could teach `morgan` how to log them differently, so we don't end up polluting logs even more?

Reply:
It would be great if we could tweak morgan to output `terminated` or `timeout` or similar to make it easy to observe, but I'm ok with splitting this out into another issue so we can get this released.